### ISSUE TITLE
Uneven Prasyarats for some MKs

### ISSUE THREAD

**Issue by chez14** [ftisunpar/data#2]
Hi,

I've tried to create [PARhunt](https://app.christianto.net/PARhunt/) to help me get a detailed MKs for next semester, but for some reason i have to [preproccess](https://gitlab.com/chez14/parhunt/blob/2c9446597fde94e1306b6be1c9ad109ac0f7cf19/src/store/appstore.js#L17) the `prasyarat.berlakuAngkatan` firsts before feeding that to my components, since on some MK it don't have the `berlakuAngkatan` property for `prasyarat`, is this intentional? Hehe.

Based on my observation, these are the MKs that don't have the `berlakuAngkatan` `prasyarat` property.
```plain
Missing Berlaku Angkatan for AIF182007 (Teknik Presentasi).
Missing Berlaku Angkatan for AIF182109 (Statistika untuk Komputasi).
Missing Berlaku Angkatan for AIF183337 (Topik Khusus Sistem Informasi 1).
Missing Berlaku Angkatan for AIF183106 (Proyek Informatika).
Missing Berlaku Angkatan for AIF183308 (Proyek Sistem Informasi 1).
```

Thanks

**Comment by pascalalfadian** [comment]
Let me think about it, or we can talk about it at lunch (remind me, please). Actually `berlakuAngkatan` should not be part of Kurikulum 2018, it is part of _Transisi_ Kurikulum 2018. But I may be wrong.

**Comment by pascalalfadian** [comment]
(Hopefully) fixed in bb8c671045004847e47f96d3e0804af55dfb8682. Please try again (with modified code)
